velocity-engine-core/src/main/java/org/apache/velocity/util/introspection/MethodMap.java

```diff
@@ -58,6 +60,8 @@ public class MethodMap
 
     TypeConversionHandler conversionHandler;
 
+    private static final Method trySetAccessible = MethodUtils.getMethodObject(Method.class, "trySetAccessible");
```

Comment:
Should be uppercase, if `static final`, no?

Reply:
Yes, I was torned between how a static final variable should look like and being more obvious that this is the `trySetAccessible` method.